spec/index.html

```diff
@@ -562,7 +562,17 @@ <h3>RDF Dataset Construction</h3>
 
 <section id="privacy-considerations" class="appendix informative">
   <h2>Privacy Considerations</h2>
-  <p class="ednote">TODO</p>  
+  <p>N-Quads is used to express arbitrary application data,
```

Comment:
It's terribly jarring to have these singular format names be lexically plural, and use singular verbs. I hope this tweak will not be controversial.
```suggestion
  <p>The N-Quads format is used to express arbitrary application data,
```